client/asset/dcr/dcr.go

```diff
@@ -386,7 +385,6 @@ type baseWallet struct {
 	wallet           Wallet
 	chainParams      *chaincfg.Params
 	log              dex.Logger
-	acct             string
```

Comment:
>  Does this mean we no longer care about using a specific wallet account for dex operations?

Depends on what you mean by **we**. `baseWallet` no longer cares, but the `Wallet` implementation does. `rpcWallet` still needs the account number as an input to its constructor. It's still part of the `Config` `struct`, but I'll note that part of that `struct` will probably be split off into a separate `RPCConfig` `struct` when a built-in wallet is also an option. 

I really think it's better to leave account handling up to the implementation. For the built-in `Wallet`, for example, we're not going to ask the user to specify an account number. But I understand your concern about the account handling necessary for the privacy implementation. Looking at #1498, it appears that the account handling could also be done in the `Wallet` implementation, but that might also result in code duplication if we want to add the privacy feature to a built-in wallet later. On the flip side, for a built-in wallet privacy implementation, the user wouldn't be setting the account numbers manually. They would be internal constants. How does that change the picture?

Reply:
I think wallets having accounts and having the ability to return balances, unspent outputs, locked utxos for an account are intrinsic to all dcr wallets, whether inbuilt or external. Instead of having `Wallet` implementations carry the burden of tracking an account number (or multiple account numbers, in the case of mixing-enabled wallets) for balance tracking and outputs selection, we can continue to have the `baseWallet` handle all that while leaving `Wallet` implementations to only do basic, generic wallet stuff.

> On the flip side, for a built-in wallet privacy implementation, the user wouldn't be setting the account numbers manually. They would be internal constants. How does that change the picture?

Just as how the btc inbuilt wallet doesn't require any details from users, the inbuilt dcr wallet doesn't need to request account information in the UI. The builtin constants can be used to populate the `ExchangeWallet.{acct,mixedAcct,changeAcct}` fields when the `ExchangeWallet` is being created.